shared/types/env.ts

```diff
@@ -10,4 +10,5 @@ export interface SwitchEnvFeedbackFormBodyDto {
   feedback: string
   email: string
   rumSessionId: string
+  radio: string
```

Comment:
nit: should we name this something clearer? like `switchReason` or something

Reply:
https://github.com/opengovsg/FormSG/pull/5245/commits/215307aefcbc89beeab0ae70e456032cb73fc71f